scripts/rollup/header.js

```diff
@@ -15,14 +15,22 @@ function getProvidesHeader(hasteFinalName) {
 `;
 }
 
-function getUMDHeader(filename, reactVersion) {
+function getHeader(filename, reactVersion) {
   return `/**
- * ${filename} v${reactVersion}
+ * React v${reactVersion}
```

Comment:
I feel version more important! Might we can do it like this?  

 /**  
 * React v16.0.0-beta.5
 * react.development.js
 *
 * Copyright 2013-present, Facebook, Inc.
 * All rights reserved.
 *
 * This source code is licensed under the BSD-style license found in the
 * LICENSE file in the root directory of this source tree. An additional grant
 * of patent rights can be found in the PATENTS file in the same directory.
 *
 */